### PR TITLE
Update Visualtest - Fix #19

### LINKED ISSUE #19: Update visual regression tests to use StableRNGs.jl
Now that #17 is merged, we can update the tests from #2 to use [StableRNGs.jl](https://github.com/rfourquet/StableRNGs.jl) so that they pass.

cc @Diaga @RashikaKarki

### PR DESCRIPTION
**Pull request by RashikaKarki**

### CODE REVIEW

**Comment by RashikaKarki** [comment]
@darsnack I believe visual regression test can now pass. Could you give it a quick review.

**Comment by darsnack** [comment]
Can you look at the `visualtest-issue-19` branch in the main repo? I added a commit to fix some dependency issues, but it seems I cannot commit directly to your branch. FYI next time you should check the box to allow commits from maintainers.

**Comment by darsnack** [comment]
Bump @RashikaKarki

**Comment by RashikaKarki** [comment]
@darsnack Would make sure to check the box on other PR's. Could you review the PR? I think I have added everything from the commit you were refering to.

File: Project.toml [hunk 1]
```diff
@@ -8,6 +8,7 @@ Adapt = "79e6a3ab-5dfb-504d-930d-738a2a938a0e"
 CuArrays = "3a865a2d-5b23-5a0f-bc46-62713ec82fae"
 DataStructures = "864edb3b-99cc-5e75-8d2d-829cb0a9cfe8"
 Distributions = "31c24e10-a181-5473-b8eb-7969acd0382f"
+Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
```
Comment (darsnack) [inline Project.toml:11]: This should be under `[deps]` and `[extras]`.

**Review by darsnack — CHANGES_REQUESTED** [review]

**Pull request merged**